registry.json

```diff
@@ -1,4 +1,19 @@
-[ 
+[
+  {
+ "ticker": "EUSKL",                                                      
+ "poolId": "pool1ksfzungz56vxa9sy6hr54v5hf6qx8rmcu6734gqm3qlxucveed6", 
```

Comment:
Your pool id is : b4122e4d02a6986e9604d5c74ab2974e80638f78e6bd1aa01b883e6e 

I fix :)